admin/threads-seq

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env -S cargo +nightly --quiet -Zscript
```

Comment:
I didn't quite mean using the `-Zscript` support, but I guess it'll do? Would be nice to name it `.rs` for now so that we get syntax highlighting?

Reply:
What did you mean by rust script? Is there another way?

(I have other callers of this script -- ie, https://github.com/ctz/openssl-bench/pull/10/files )

Reply:
My suggestion was more like sticking a normal `threads-seq.rs` in an `examples` dir, without relying on nightly script support.

Reply:
I think I prefer the -Zscript way, and AIUI it will stop needing nightly very very soon.